.github/workflows/build.yml

```diff
@@ -0,0 +1,112 @@
+name: 'confluent-kafka-dotnet build pipeline'
+
+env:
+  CONFIGURATION: Release
+  DOTNET_CLI_TELEMETRY_OPTOUT: 'true'
+
+on:
+  push:
+  pull_request:
+
+jobs:
+
+  linux-build:
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v4
+      - name: Setup .NET
+        uses: actions/setup-dotnet@v3
+        with:
+          dotnet-version: '6.0.x'
```

Comment:
Why this old version of .NET everywhere, can we use a more recent SDK version? Also in some places you've specified `6.0.x` and in others `6.0.403`, is that on purpose?

Reply:
I added .net8.0.x to the list (the library targets `netstandard2.0;net462;net6.0;net8.0`). It worked before, as .NET is already installed on the runners by default.

Reply:
Ah right, I hadn't realised that. Thanks